pkgs/by-name/sp/spec-kit/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```
a bit easier to override

---

nuh-uh, i referred to all instances of `python3.pkgs` in this file (also adjust the top args too)